packages/api/src/schema/api-schema.yaml

```diff
@@ -1890,15 +1966,28 @@ components:
                     cid:
                       type: string
                       description: IPFS CID of the exported data
+                      example: Qmabc123xyz341
+
     input-creator-id:
       oneOf:
-        - $ref: "#/components/schemas/creator-id"
```

Comment:
Can we keep the inline `$ref` given the fix on `compile-schemas.js`? Or is this part of the fix?